kernel/include/ext2.h

```diff
@@ -143,7 +143,10 @@ struct ext2_inodetable {
 	uint32_t blocks;
 	uint32_t flags;
 	uint32_t osd1;
-	uint32_t block[15];
+	union {
```

Comment:
There's also the option of using a named union named something like sb (symlink/block). Makes getting at the block require inode->sb.block, but eh.